src/app/modules/dashboard/dashboard.component.html

```diff
@@ -1,10 +1,16 @@
 <div class="grid">
+  <ng-container *ngFor="let item of dashboardStats | keyvalue">
```

Comment:
There should be a `ngIf` condition for `null/undefined` checking for `dashboardStats`